lms/tasks/email_digests.py

```diff
@@ -1,13 +1,67 @@
 import logging
-from datetime import datetime
+from datetime import datetime, timedelta, timezone
 from typing import List, Optional
 
+from sqlalchemy import select
+
+from lms.models import ApplicationInstance, AssignmentMembership, LTIRole, User
 from lms.services import DigestService
 from lms.tasks.celery import app
 
 LOG = logging.getLogger(__name__)
 
 
+@app.task
+def send_instructor_email_digest_tasks(batch_size):
```

Comment:
This function name is very similar to the name of the function below that it calls (`send_instructor_email_digests()`) but I couldn't think what else to call it

Reply:
Making `batch_size` an argument means that the actual `batch_size` will be set in `h-periodic` where it's easy to change (`h-periodic` is really quick to deploy).